pytx/pytx/vocabulary.py

```diff
@@ -5,7 +5,7 @@ class ThreatExchange(object):
     """
 
     URL = 'https://graph.facebook.com/'
-    VERSION = 'v2.6/'
+    VERSION = 'v2.8/'
```

Comment:
This was something I was considering while developing this PR. I had intentions of removing `attack_type` and `threat_type` from the code base as well as bumping to `0.6.0` and updating the documentation.

However, after looking at the `2.8` developer documentation I noticed that `attack_type` and `threat_type` are still referenced as valid fields. Also, when submitting queries, you can still request and see the values for those fields in the response data. As a side note didn't see any `tags` for the returned objects which I was expecting to at least have the `attack_type` and `threat_type` there as part of the migration effort to deprecate the `attack_type` and `threat_type`.

I'm open to changing the code to better reflect how we expect the API to be used, just let me know!